src/Keyborg.ts

```diff
@@ -183,7 +183,7 @@ class KeyborgCore implements Disposable {
       return;
     }
 
-    const details = e.details;
+    const details = e.detail;
```

Comment:
This will break Tabster as it uses `details`. I was probably not named according to the CustomEvent signature, but it is hard to rename it now.

Reply:
makes sense - I've added the `event.details` field again under `ts-ignore`